tests/test_services.py

```diff
@@ -147,6 +175,36 @@ def test_unwrap_invalid_char(service):
         "CurrentLEDState": "On",
         "Unicode": "AB"}
 
+
+def test_compose(service):
+    """Test argument composition."""
+    service.DEFAULT_ARGS = {}
+
```

Comment:
Can you throw a few scattered comments is here about what it is that the different sections of test code tests. I mean, I can read it, if I scroll back and forth, between the test code and the test data, but I think it could use just a few small comments.